testbed/.gitignore

```diff
@@ -0,0 +1,13 @@
+*.js
+!jest.config.js
+*.d.ts
+node_modules
+.DS_Store
```

Comment:
I prefer to keep .gitignores minimal and intentional and specific to the files necessary for the project (e.g. node_modules should be in here). 

In general, it's best practice to have a developer global ~/.gitignore file that ignores things like IDE settings and OSX finder registries (.vscode, .DS_STORE). Otherwise, the .gitignore must contain the union set of every developers environment, which can get out of hand.